dice_ml/explainer_interfaces/dice_random.py

```diff
@@ -138,7 +132,13 @@ class of query_instance for binary classification.
             if len(cfs_df) > total_CFs:
                 cfs_df = cfs_df.sample(total_CFs)
             cfs_df.reset_index(inplace=True, drop=True)
-            self.cfs_pred_scores = self.predict_fn(cfs_df)
+            if len(cfs_df) > 0:
+                self.cfs_pred_scores = self.predict_fn(cfs_df)
+            else:
```

Comment:
seems arbitrary to assign zero to the `cfs_pred_scores`. Is None a better alternative?